mautic/mautic
# Fixing global search JS not be flash irrelevant message.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ✅ ]
| New feature/enhancement? (use the a.x branch)      | [ :x:]
| Deprecations?                          | [ :x:]
| BC breaks? (use the c.x branch)        | [ :x:]
| Automated tests included?              | NA
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #11977 

#### Description:

Global Search will sometimes give errors but will continue to work.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Log in to the mautic
3. Do a global search
